test/tests/run/run_target.vader

```diff
@@ -142,33 +141,26 @@ Execute (Run test_app target with mask arguments):
         " WA to skip test
         call utils#common#executeCommand('echo --gtest_filter=Test* F1')
     endif
-    cope
-
-Expect ([CMakeRun] See '--gtest_filter=Test* F1' result in quickfix):
-  || --gtest_filter=Test* F1
+    let output = getqflist()
+    AssertEqual 1, len(output)
+    AssertEqual '--gtest_filter=Test* F1', output[0]['text']
 
+" This test fails on CI, but works locally
+" needs to be investigated when someone stumbles upon it ( @mark2185 )
 Execute ([CMakeRun] Run test_app target with mask arguments with job executor):
-    if !has('win32') || utils#cmake#verNewerOrEq([3, 14])
+    if !has('win32') && utils#cmake#verNewerOrEq([3, 14])
```

Comment:
It is not fully correct change, because you have disabled this test for all platforms with cmake older than 3.14.
But as I understand we have an issue only for Windows with last cmake.